crates/wysiwyg/src/dom/nodes/text_node.rs

```diff
@@ -128,10 +126,7 @@ where
         direction: &Direction,
     ) -> Option<CharType> {
         let char = self.char_at_offset(offset, direction);
-        match char {
-            Some(c) => Some(get_char_type(c)),
-            None => None,
-        }
+        char.map(get_char_type)
```

Comment:
I thought there must have been a better place to express this when I wrote it

Reply:
@alunturner  You can add `"rust-analyzer.checkOnSave.command": "clippy",` in e.g. VS Code json settings, that's what point me towards this.